<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-IEK94JO" ignored="0" name="test.AddToCartTest" tests="5" failures="1" timestamp="22 черв. 2022 21:38:30 GMT" time="80.166" errors="0">
  <testcase name="verifyThatCartContainsBillThatIsLessThanSpecifiedSum" time="17.292" classname="test.AddToCartTest"/>
  <testcase name="verifyThatCartContainsBillThatIsLessThanSpecifiedSum" time="8.587" classname="test.AddToCartTest"/>
  <testcase name="verifyThatCartContainsBillThatIsLessThanSpecifiedSum" time="8.931" classname="test.AddToCartTest"/>
  <testcase name="verifyThatCartContainsBillThatIsLessThanSpecifiedSum" time="8.692" classname="test.AddToCartTest"/>
  <testcase name="verifyThatCartContainsBillThatIsLessThanSpecifiedSum" time="8.982" classname="test.AddToCartTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.AddToCartTest.verifyThatCartContainsBillThatIsLessThanSpecifiedSum(AddToCartTest.java:30)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:750)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- verifyThatCartContainsBillThatIsLessThanSpecifiedSum -->
</testsuite> <!-- test.AddToCartTest -->
